modules/admin_manual/pages/configuration/server/occ_commands/core_commands/_previews_commands.adoc

```diff
@@ -0,0 +1,58 @@
+= Previews Commands
+
+A set of commands to remove unreferenced preview images:
+
+[source,plaintext]
+----
+previews
+ previews:cleanup            Remove unreferenced previews
+----
+
+== Cleanup
+
+Removing not referenced previews can be necessary, e.g., when the image has been deleted or the ID of the mountpoint changes which happens when renaming it. Previews will be generated anew when accessing the image again, but not referenced previews are orphans. To avoid orphans, an occ command has been added to manually remove those not referenced previews as they are eating up storage space unnecessarily. In addition, a background job has been created which automatically removes them on a regular basis. For more info see the xref:background-job[Background Job] section and the example listing xref:configuration/server/occ_command.adoc#list-queued-backgroundjobs[OCA\Files\BackgroundJob\PreviewCleanupJob].
+
+[source,bash,subs="attributes+"]
+----
+{occ-command-example-prefix} previews:cleanup [options] [--] [<chunk_size>]
+----
+
+=== Arguments
+
+[width="100%",cols="25%,70%",]
+|====
+| `chunk_size`
+| Define the number of files being processed in one loop [default: 1000]
+|====
+
+=== Options
+
+[width="100%",cols="25%,70%",]
+|====
+| `-all`
```

Comment:
```suggestion
| `--all`
```

Reply:
@mmattel I commented a few seconds late - can you make this fix in another PR please?

Reply:
Will fix in master, fixed it in the backport already...